panoptes_client/collection.py

```diff
@@ -12,7 +13,13 @@ class Collection(PanoptesObject):
     _edit_attributes = (
         'name',
         'display_name',
-        'private'
+        'private',
+        {
+            'links': (
+                'project',
+                'default_subject',
+            ),
+        },
```

Comment:
I would revert this change, since these attributes shouldn't be edited directly.